tmt/steps/provision/__init__.py

```diff
@@ -778,7 +778,7 @@ def _random_name(self, prefix: str = '', length: int = 16) -> str:
         # Append at least 5 random characters
         min_random_part = max(5, length - len(prefix))
         name = prefix + ''.join(
-            random.choices(string.ascii_letters, k=min_random_part))
+            secrets.choice(string.ascii_letters) for _ in range(min_random_part))
```

Comment:
Feels like there can be a simpler approach here.

Reply:
felt to me too, but this is basically "best practices" from python secrets documentation and was suggested by @thrix (and I haven't found anything better fwiw)